modules/olmv1-updating-an-operator.adoc

```diff
@@ -159,6 +161,53 @@ spec:
 ----
 <1> Update the version from `1.11.1` to `1.12.1`
 
+** If you want to define a range of acceptable update versions, edit your CR similar to the following example:
++
+.Example CR with a version range specified
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  version: >1.11.1, <1.13 <1>
+----
+<1> Specifies that the desired version range is greater than version `1.11.1` and less than `1.13`. For more information, see "Support for version ranges".
+
+** If you want to update to the latest version that can be resolved from a channel, edit your CR similar to the following example:
++
+.Example CR with a specified channel
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  channel: latest <1>
+----
+<1> Installs the latest release that can be resolved from the specified channel. Updates to the channel are automatically installed
+
+** If you want to specify a channel and version or version range, edir your CR similar to the following example:
++
+.Example CR with a specified channel
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  channel: latest
+  version: <1.13
+----
++
+For more information, see "Example custom resources that specify a target version".
```

Comment:
Can we also embed the link here too?
- #olmv1-about-target-versions_olmv1-installing-operator